src/styles/card.scss

```diff
@@ -0,0 +1,157 @@
+.container {
+  display: grid;
+  justify-content: center;
+  grid-template-columns: repeat(var(--column-count), 200px);
+  padding: 50px 48px 0;
+
+  margin-bottom: 72px;
+  grid-row-gap: 46px;
+  column-gap: 48px;
+}
+
+@media (min-width: 0) {
+  .container {
+    --column-count: 1;
+  }
+}
+
+@media (min-width: 488px) {
+  .container {
+    --column-count: 2;
+  }
+}
+
+@media (min-width: 768px) {
+  .container {
+    --column-count: 3;
+  }
+}
+
+@media (min-width: 1024px) {
+  .container {
+    --column-count: 4;
+  }
+}
+
+.card {
+  display: flex;
+  flex-direction: column;
+  width: 200px;
+  height: 408px;
+  padding: 32px 16px 16px;
+
+  border-radius: 5px;
+  border: 1px solid var(--border-color);
+  background: var(--button-color-hover);
+  transition: all 0.3s ease-in-out;
+
+
+  &__title {
+    margin-top: 0;
```

Comment:
More correctly reset default margin directly for h2, h3..., p tags instead of resetting them in every class